src/dbg/debugger.cpp

```diff
@@ -280,9 +280,9 @@ bool dbgcmddel(const char* name)
     return true;
 }
 
-duint dbggetdbgevents()
+uint64 dbggetdbgevents()
 {
-    return InterlockedExchange(&DbgEvents, 0);
+    return DbgEvents;

```

Comment:
race condition when reading out the value. Use an interlocked function to read instead.